packages/core-data/src/actions.js

```diff
@@ -441,11 +443,18 @@ export function* saveEntityRecord(
 								: data.status,
 					}
 				);
-				updatedRecord = yield apiFetch( {
+				const request = {
 					path: `${ path }/autosaves`,
 					method: 'POST',
 					data,
-				} );
+				};
+				if ( __experimentalBatch ) {
+					updatedRecord = yield __unstableAwaitPromise(
```

Comment:
Not sure why `__unstableAwaitPromise` is marked unstable.

Reply:
Also, thinking it might be more "rungen-ey" to make this a dedicated control.

```js
updatedRecord = yield addToBatch( batch, request );
```

Reply:
> Not sure why __unstableAwaitPromise is marked unstable.

I think it was initially merged with the locks PR, and all the APIs there were marked as unstable. Feel free to "stabilize" it :-)

Reply:
Going to keep it unstable because I hope we can remove it if https://github.com/WordPress/gutenberg/pull/27276 lands.